src/djcytoscape/models.py

```diff
@@ -15,6 +15,29 @@
 from quest_manager.models import Category
 
 
+def validate_url(url):
```

Comment:
I'm pretty sure we are already doing this elsewhere in the app.  It doesn't make sense to have to maintain two versions of a local url (relative path) validator.  Can you take a look around and see if you can find it? I may be imagining things.

If you do find it, pick the best implementation (or extend one if the needs are not exactly the same)

Reply:
Alternately, can we just USE the flatpages validator mentioned in the docstring?

Reply:
![image](https://github.com/user-attachments/assets/006820f1-70aa-418a-9882-3c84736ea6fe)
![image](https://github.com/user-attachments/assets/81148c44-2de0-44a1-94f8-49c6b3107925)

flatpages doesnt have its own validator. Im pretty sure it would take more work to jury-rig flatpage form directly than what currently exist

Reply:
found an alternative `URLOrRelativeURLField`

Reply:
not entirely sure to keep the validate url tests. `URLOrRelativeURLField` is basically `URLField` but if it starts with a `/` it adds `www.example.com` at the start